core/templates/dev/head/services/RteHelperService.js

```diff
@@ -89,6 +90,10 @@ oppia.factory('RteHelperService', [
             explorationId: ExplorationContextService.getExplorationId()
           });
         }
+        customizationArgsDict = angular.extend(customizationArgsDict, {
+          urlWithBucket: ('https://storage.googleapis.com/' +
+            GLOBALS.GCS_RESOURCE_BUCKET_NAME)
```

Comment:
Have just bucketName here and can you move this to the line 91, so it would be together with the explorationId add?

Reply:
But there is a condition for that `if (ExplorationContextService.isInExplorationContext())`.

I think it makes sense to shift it to line 91 because anyways, exploratiion_id is needed, so the condition has to satisfy for it.
What do you say ?

Reply:
Yes, if explorationId is added we can assume buckeId would be needed too, so it is logical to move it there.